src/test/java/org/opentripplanner/middleware/auth/Auth0ConnectionTest.java

```diff
@@ -0,0 +1,55 @@
+package org.opentripplanner.middleware.auth;
+
+import com.auth0.exception.Auth0Exception;
+import com.auth0.json.auth.TokenHolder;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class Auth0ConnectionTest {
+
+    static TokenHolder cachedToken = null;
+
+    @BeforeAll
+    public static void setUp() throws IOException {
+        cachedToken = new ObjectMapper().readValue("{\"expires_in\": 86400}", TokenHolder.class);
+    }
+
+    /**
+     * Shows that TokenWrapper.getExpiresIn() is invariant
+     * (see https://github.com/auth0/auth0-java/blob/master/src/main/java/com/auth0/json/auth/TokenHolder.java)
+     * it is the duration of the token validity.
+     */
+    @Test
+    public void tokenExpiryIsActuallyTokenLifetime() throws Auth0Exception, InterruptedException {
```

Comment:
Auth0Exception is never thrown and can be removed.

Reply:
Test has been removed.